Umbraco-Cloud/Set-Up/Project-settings/index.md

```diff
@@ -5,7 +5,7 @@ versionTo: 10.0.0
 
 # Project Settings
 
-When working with a Umbraco Cloud project, you can handle a lot of the project configuration directly in the Umbraco Cloud Portal. You can manage team members, database connections, hostnames/domains, security settings, SSL certificates, deployment webhooks, upgrade plan, project usage, advanced settings, rename, or delete the project.
+When working with a Umbraco Cloud project, you can handle a lot of the project configuration directly in the Umbraco Cloud Portal. You can manage team members, database connections, hostnames/domains, security settings, Secure Sockets Layer (SSL) certificates, deployment webhooks, upgrade plan, project usage, advanced settings, rename, or delete the project.
```

Comment:
How about taking the last sentence here, and making a list instead?

Reply:
Done